app/javascript/components/analysis/selectors.js

```diff
@@ -28,6 +28,8 @@ const selectEmbed = state =>
   state.location.pathname.includes('/embed');
 const selectError = state => state.analysis && state.analysis.error;
 const selectDatasets = state => state.datasets && state.datasets.data;
+const selectGeostoreSize = state =>
+  state.geostore && state.geostore.data && state.geostore.data.areaHa;
```

Comment:
not saying you should use it, just as a curiosity, there's a lodash `get` helper that lets you access deeply nested props (and if one of them at some point is `null` or `undefined` it'll return undefined): https://masteringjs.io/tutorials/lodash/get

Reply:
I didn't know about that! Very cool. I think as we are doing the same across the site then not worth changing now, but will try to start using. The babel 7 updates for ?. chaining I guess do the same thing.